lib/common/metric/BaseMetricFactory.ts

```diff
@@ -1,15 +1,33 @@
 import { MetricFactory } from "./MetricFactory";
 
 export interface BaseMetricFactoryProps {
-  // TODO: this will eventually include other common things like account/region
+  /**
+   * Region where the metrics exist.
+   *
+   * @default The region configured by the construct holding the Monitoring construct
```

Comment:
Will this default be overridden by the one defined on the facade? We should probably mention the order (segment > facade > construct)?